src/components/ImageUploader/src/ImageSelection.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+	<div
+		:class="{
+			[$s.ImageSelectionContainer]: true,
+			[$s.ImageSelectionContainerError]: isError
+		}"
+		role="img"
+		:style="bgImageStyle"
+	>
+		<div
+			v-if="isUploading"
+			:class="$s.ImageSelectionLoaderContainer"
+		>
+			<loading :class="$s.ImageSelectionLoader" />
+		</div>
+
+		<progress-bar
+			:class="$s.ImageSelectionProgressContainer"
+			:style="progressContainerStyle"
+			:progress="progress"
+			size="xsmall"
+		/>
+
+		<button
+			v-if="!isUploading"
+			type="button"
+			:class="$s.ImageSelectionRemoveButton"
+			@click="$emit('removeImage')"
+		>
+			<x-icon
+				inline
+				:class="$s.ImageSelectionRemoveIcon"
+			/>
+		</button>
+	</div>
+</template>
+
+<script>
+import { MLoading as Loading } from '@square/maker/components/Loading';
+import { MProgressBar as ProgressBar } from '@square/maker/components/ProgressBar';
```

Comment:
Why import them with a different name? The `M` prefix on the component will help clarify which components they are when others see them in the template.

Reply:
I had misunderstood another change request above, thought they meant all maker components. But I've reverted it.